<ui version="4.0" >
 <class>WhiteBoard</class>
 <widget class="QWidget" name="WhiteBoard" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>@product@ - Video</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>0</number>
   </property>
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame" >
     <property name="minimumSize" >
      <size>
       <width>176</width>
       <height>144</height>
      </size>
     </property>
     <property name="styleSheet" >
      <string notr="true" >background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="3" >
       <widget class="QToolButton" name="rectButton" >
        <property name="text" >
         <string>矩形</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4" >
       <widget class="QToolButton" name="eillButton" >
        <property name="text" >
         <string>圆形</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5" >
       <widget class="QToolButton" name="handButton" >
        <property name="text" >
         <string>手动</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6" >
       <widget class="QToolButton" name="undoButton" >
        <property name="text" >
         <string>撤销</string>
        </property>
       </widget>
      </item>
      <item row="0" column="7" >
       <widget class="QToolButton" name="redoButton" >
        <property name="text" >
         <string>重画</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="12" >
       <layout class="QGridLayout" name="gridLayout_2" >
        <item row="0" column="0" >
         <widget class="QScrollArea" name="scrollArea" >
          <property name="widgetResizable" >
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents" >
           <property name="geometry" >
            <rect>
             <x>0</x>
             <y>0</y>
             <width>561</width>
             <height>343</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="8" >
       <widget class="QToolButton" name="clearButton" >
        <property name="text" >
         <string>清空</string>
        </property>
       </widget>
      </item>
      <item row="0" column="11" >
       <spacer name="horizontalSpacer" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1" >
       <spacer name="horizontalSpacer_2" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2" >
       <widget class="QToolButton" name="lineButton" >
        <property name="text" >
         <string>直线</string>
        </property>
       </widget>
      </item>
      <item row="0" column="9" >
       <widget class="QToolButton" name="saveButton" >
        <property name="text" >
         <string>保存</string>
        </property>
       </widget>
      </item>
      <item row="0" column="10" >
       <widget class="QToolButton" name="openButton" >
        <property name="text" >
         <string>打开</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
